compiler-plugin/src/main/java/io/ballerina/stdlib/sql/compiler/Utils.java

```diff
@@ -81,7 +85,15 @@ public static boolean isSQLOutParameter(TypeReferenceTypeSymbol typeReference) {
     }
 
     public static DiagnosticInfo addDiagnosticsForInvalidTypes(String outParameterName,
-                                                               TypeDescKind requestedReturnType) {
+                                                               TypeSymbol argumentTypeSymbol) {
+        TypeDescKind requestedReturnType = argumentTypeSymbol.typeKind();
+        String requestedReturnTypeName = "";
```

Comment:
```suggestion
        String requestedReturnTypeArgValue = "";
```

Reply:
Fixed in [2c8f63d](https://github.com/ballerina-platform/module-ballerina-sql/pull/520/commits/2c8f63d0c9c7ef0ed89521af055fe0a8fd4839d1)